[NO NEW TESTS NEEDED] add builddeps to copr template

lsm5 opened the pull request:
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

The copr build failed cause the build deps weren't added to the template. This commit should fix that.

@baude @mheon @vrothberg @edsantiago @cevich PTAL

/cc @containers/podman-maintainers 

<!--
Thanks for sending a pull request!

Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).

In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.

Finally, be sure to sign commits with your real name.  Since by opening
a PR you already have commits, you can add signatures if needed with
something like `git commit -s --amend`.
-->

vrothberg reviewed (commented):
LGTM

mheon commented:
/lgtm
/hold

edsantiago commented:
I missed the initial rpkg PR (#12868) and I don't really have any idea what this is for. Should I diff against `contrib/spec/podman.spec`? Against the one in `fedpkg`? I see lots of differences, such as in missing `plugins`, `LDFLAGS`, `GO111MODULE`, and too many to list.

It really disturbs me to have so many different inconsistent specfiles all over the place. Is there any way we could autogenerate this one (via `sed`, or `m4`, or something) from one of the already-existing specfiles?

mheon commented:
Is the one in `contrib/spec/podman.spec.in` still in use?

Can we nuke it and replace with this?

lsm5 commented:
@edsantiago @mheon sorry, The diff should be against the file deleted in #12868 . I removed the `.autocopr/podman.spec` and replaced it with this template. The other file `contrib/spec/podman.spec` I think i used somewhere else. I'll file a PR to get rid of that later but that doesn't affect this PR. HTH.

lsm5 commented:
i added a dependency on netavark. PTAL once again. Should be good to go now.

rhatdan commented:
/lgtm
/hold cancel
